test/commands/test_alias.rb

```diff
@@ -1,7 +1,48 @@
 require 'minitest/autorun'
+require_relative '../test__helper'
+require_relative '../fake_home'
+require_relative '../../lib/zold/commands/alias'
 
-# @todo #279:30min Write unit tests for the alias command.
-#  It should test all the subcommands (set, remove and show)
-#  and any options.
 class TestAlias < Minitest::Test
+  # alias set <wallet> <alias>
+  def test_set_writes_alias_to_the_alias_file
+    skip
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      Zold::Alias.new(wallets: home.wallets, log: test_log).run(%W[set #{wallet.id} my-alias])
+      assert_equal read_alias_file(home), %W[my-alias #{wallet.id}]
+    end
+  end
+
+  # alias remove <alias>
+  def test_remove_removes_the_alias_from_the_alias_file
+    skip
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      cmd = Zold::Alias.new(wallets: home.wallets, log: test_log)
+      cmd.run(%W[set #{wallet.id} my-alias])
+      assert_equal read_alias_file(home), %W[my-alias #{wallet.id}]
+      cmd.run(%w[remove my-alias])
+      assert_empty read_alias_file(home)
+    end
+  end
+
+  # alias show <alias>
+  def test_show_prints_out_the_aliased_wallet_id
```

Comment:
@botanicus you will probably need puzzles, otherwise the tests won't be unskipped.

Reply:
@ledestin all the methods have puzzles already (https://github.com/zold-io/zold/pull/331/files#diff-84261a33fb15ff5868db815f7323358fR37).

You're right though, it should be part of one PR, but we created the puzzles in a not very logical way before, having #323 and #322, so now it's hard to do anything about it.